src/commands/NewIntegrationFileCommand.ts

```diff
@@ -32,27 +32,50 @@ const LANGUAGES_WITH_FILENAME_EXTENSIONS = new Map([
 	['Kotlin', 'kts']]);
 const LANGUAGES = Array.from(LANGUAGES_WITH_FILENAME_EXTENSIONS.keys());
 
-export async function create() : Promise<void> {
-	const language = await vscode.window.showQuickPick(LANGUAGES, {placeHolder:'Please select the language in which the new file will be generated.'});
-	if (language) {
-		const workspaceFolder = await vscode.window.showWorkspaceFolderPick(
-			{placeHolder: 'Please select the workspace folder in which the new file will be created.'});
-		if (workspaceFolder) {
-			const filename = await vscode.window.showInputBox({
-				prompt: 'Please provide a name for the new file (without extension)',
-				validateInput: (name: string) => {
-					return validateFileName(name, language, workspaceFolder);
-				}
-			});
-			if (filename) {
-				const kamelExe = kamel.create();
-				const newFileFullPath: string = computeFullpath(language, workspaceFolder, filename);
-				await kamelExe.invoke(`init "${newFileFullPath}"`);
-				const textDocument = await vscode.workspace.openTextDocument(newFileFullPath);
-				await vscode.window.showTextDocument(textDocument);
+export async function create(...args: any[]) : Promise<void> {
+
+	let language : string | undefined;
+	let workspaceFolder : vscode.WorkspaceFolder | undefined = undefined;
```

Comment:
I think that `undefined` is the default anyways when not assigning a value so it could be removed.

Reply:
I can try it but I seem to remember running into some weirdness when I didn't explicitly specify `string | undefined`

Reply:
I meant changing it to `let workspaceFolder : vscode.WorkspaceFolder | undefined;`